libs/ui/select/brain/src/lib/brn-select-trigger.directive.ts

```diff
@@ -0,0 +1,38 @@
+import { Directive, ElementRef, OnDestroy, computed, inject } from '@angular/core';
+import { Subject } from 'rxjs';
+import { BrnSelectService } from './brn-select.service';
+
+@Directive({
+	selector: '[brnSelectTrigger]',
+	standalone: true,
+	host: {
+		role: 'combobox',
+		'[attr.id]': 'selectTriggerId()',
+		'[disabled]': 'selectDisable()',
+		'[attr.aria-expanded]': 'isExpanded()',
+		'[attr.aria-controls]': "selectContentId() + ''",
+	},
+})
+export class BrnSelectTriggerDirective implements OnDestroy {
+	unsubscribe = new Subject<boolean>();
```

Comment:
unused code?

Reply:
I think I was preparing to subscribe to something but then just didn't end up doing it that way haha but yes good eyes @elite-benni